core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/java/org/fusesource/ide/camel/model/service/core/tests/integration/core/model/CamelBasicModelElementTestIT.java

```diff
@@ -0,0 +1,61 @@
+/******************************************************************************* 
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.fusesource.ide.camel.model.service.core.tests.integration.core.model;
+
+import static org.fusesource.ide.preferences.PreferencesConstants.EDITOR_PREFERRED_LABEL;
+import static org.junit.Assert.assertEquals;
+
+import org.fusesource.ide.camel.model.service.core.catalog.eips.Eip;
+import org.fusesource.ide.camel.model.service.core.model.AbstractCamelModelElement;
+import org.fusesource.ide.camel.model.service.core.model.CamelBasicModelElement;
+import org.fusesource.ide.preferences.PreferenceManager;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * 
+ * @author Andrej Podhradsky (apodhrad@redhat.com)
+ *
+ */
+public class CamelBasicModelElementTestIT {
+
+	@Before
+	@After
+	public void cleanUserLabelsPreference() {
+		PreferenceManager.getInstance().savePreference(EDITOR_PREFERRED_LABEL, "");
```

Comment:
you are nto sure to let the systeùm in teh same state like that.
You need to retrieve the value in the @Before and then set it back in @After